docs/install/README.md

```diff
@@ -3,14 +3,14 @@
 To run you own project "Steward" you need a Kubernetes cluster.
 Currently **Kubernetes 1.14** is recommended.
 
-## Install Tekton v0.8.0
+## Install Tekton v0.15.2
```

Comment:
0.15.2 or 0.14.3?

Reply:
I tried first with 0.15.2 and then switched to 0.14.3. 
This is fixed in the README now.